CMakeLists.txt

```diff
@@ -312,10 +329,11 @@ elseif (APPLE)
 	endif ()
 
     if (BUILD_PYTHON)
-	    set (CMAKE_INSTALL_RPATH "@loader_path")
-	else ()
-	    #set (CMAKE_INSTALL_RPATH "@executable_path/../lib")
-	    set (CMAKE_INSTALL_RPATH "@executable_path/../Frameworks")
+        set (CMAKE_INSTALL_RPATH "@loader_path")
+    elseif ("${CMAKE_OSX_ARCHITECTURES}" STREQUAL "arm64")
+        set (CMAKE_INSTALL_RPATH "@executable_path/../Frameworks;@executable_path/../Resources")
```

Comment:
RPATH is for finding libraries/frameworks which for an AppBundle should exist inside the Frameworks directory. Is there a reason we also need it to link to the Resources directory?

From the macOS docs:
```
Frameworks: Contains any private shared libraries and frameworks used by the executable
Resources: data files that live outside your application’s executable file. Resources typically consist of things like images, icons, sounds, nib files, strings files, configuration files, and data files
```

Reply:
Yes, it's a codesigning requirement.  Frameworks cannot contain non-binary files like .py modules, so Python needs to reside in Resources.

Reply:
Just a followup with a reference.  This is also how Blender distributes Python.

> Store Python, Perl, shell, and other script files and other non-Mach-O executables in your app's Contents/Resources directory. While it's possible to sign such executables and store them in Contents/MacOS, this is not recommended.

https://developer.apple.com/library/archive/technotes/tn2206/_index.html#//apple_ref/doc/uid/DTS40007919-CH1-TNTAG13

Reply:
I see the issue and I am fine keeping it this way.

To respond to your followup:

> Store Python, Perl, shell, and other script files and other **non-Mach-O executables** in your app's Contents/Resources directory. While it's possible to sign such executables and store them in Contents/MacOS, this is not recommended.

Blender's macOS app does not follow the specification at all.

Reply:
Ok, thanks, but for the record:

> Blender's macOS app does not follow the specification at all.

Blender puts all of its libraries in the Resources directory.  Could you elaborate if I'm mistaken?

Reply:
Correct, which directly contradicts the specification.